### PR TITLE
Force pagination in history API endpoint

### PR DESCRIPTION
**Pull request by kantai**
This adds a `page=` query string to the API endpoint for history, and forces it's use (as in defaults to page=0).

### CODE REVIEW

**Comment by jcnelson** [comment]
Looks good @kantai!  I think that as long as the `page=` parameter isn't required, and that it returns the most recent N history items (which should already be the case), then I'm not too worried about breakage.  We should nevertheless open an issue on the explorer to use pagination properly, once this ships.

File: blockstack_client/rpc.py [hunk 1]
```diff
@@ -1092,6 +1092,7 @@ def GET_name_history(self, ses, path_info, name ):
         qs_values = path_info['qs_values']
         start_block = qs_values.get('start_block', None)
         end_block = qs_values.get('end_block', None)
+        page = int(qs_values.get('page', 0))
```
Comment (jcnelson) [inline blockstack_client/rpc.py]: Can we require that either `start_block` and `end_block` are present, or that `page` is present, but not both?  Also, we should think about deprecating `start_block` and `end_block` in favor of pagination.

**Comment by kantai** [comment]
Yep -- if the page parameter isn't passed, it just returns page 0, which is the latest 20 events. Updated the branch to master (because the previous base was just merged) -- let me know if this is fine, and I'll merge.

**Comment by jcnelson** [comment]
Fine by me.  Go ahead and merge :)

**Pull request merged**